app/src/main/res/layout/content_result.xml

```diff
@@ -8,29 +8,47 @@
     tools:context=".Result"
     tools:showIn="@layout/activity_result">
 
-    <ImageView
-        android:layout_width="match_parent"
+    <EditText
+        android:id="@+id/editTextTextPersonName"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:adjustViewBounds="true"
-        android:src="@drawable/ic_launcher_background"
+        android:layout_marginStart="100dp"
+        android:layout_marginEnd="101dp"
+        android:layout_marginBottom="4dp"
+        android:ems="10"
+        android:inputType="textPersonName"
+        android:text="Results"
+        android:textAlignment="center"
+        android:textAllCaps="true"
+        app:layout_constraintBottom_toTopOf="@+id/label"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent"

```

Comment:
These 2 confuses me. I am unsure how the text is not flailing all over the place since you technically constrained it to the parent stretching from the top of the screen to the bottom of the screen. I need to double check how this works again myself.